charts/httpbin/Chart.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v2
+name: httpbin
+description: A Helm chart for Kubernetes
```

Comment:
Should change this to something like "Official Helm chart for the httpbin service"